sdk/keyvault/keyvault-secrets/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 4.0.3 (2020-05-11)
```

Comment:
It is unlikely that we will get to ship the hot fix today.
The core-tracing and core-http dependency updates are yet to be done and we need to allocate some time to go through the release check list. I would recommend we do the release tomorrow.

Reply:
I'll update the dates!

Reply:
I think it makes more sense to move the date to the 13th. I fear the changes here https://github.com/Azure/azure-sdk-for-js/pull/7998 will take me some more time than what's expected, since upgrading these versions will need several updates and fixes, some of which might not be expected, since the base branch is considerably different.

Reply:
CI is also getting out of memory and hitting timeouts. I know CI has been changed over time to be more strict on the timeout, our tests have adapted too, but these original tests could be too old to fit.